rwengine/src/engine/SaveGame.cpp

```diff
@@ -1313,13 +1313,18 @@ bool SaveGame::getSaveInfo(const std::string& file, BasicState* basicState) {
 std::vector<SaveGameInfo> SaveGame::getAllSaveGameInfo() {
     using namespace boost::filesystem;
 
-    // TODO consider windows
+#if defined(RW_WINDOWS)
+    auto homedir = getenv("USERPROFILE");
+    const char gameDir[] = "\\My Documents\\GTA3 User Files";
```

Comment:
For Windows, I would suggest using [SHGetKnownFolderPath](https://msdn.microsoft.com/en-us/library/windows/desktop/bb762188(v=vs.85).aspx) and [FOLDERID_SavedGames](https://msdn.microsoft.com/en-us/library/windows/desktop/dd378457(v=vs.85).aspx).